File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    int a = 0,num_coin = 0;
    float amount = 0;

    do
    {
        amount = get_float("Change owed: ");
    }
    while (amount < 0);

        a = round(amount*100);

        int quarter = a / 25;
        int dime = (a % 25) / 10;
        int nickel = ((a % 25) % 10) / 5;
        int pennie = ((a % 25) % 10) % 5;
        num_coin = quarter + dime + nickel + pennie;

    printf("%d\n",num_coin);
}

File: pset6/cash/cash.py
import cs50


def main():

    while True:
        dollars_owed = cs50.get_float("Change owed: ")

        if dollars_owed > 0:
            break

    num_of_coins = 0
    cents = int(round(dollars_owed * 100))

    num_of_coins += cents // 25
    cents %= 25

    num_of_coins += cents // 10
    cents %= 10

    num_of_coins += cents // 5
    cents %= 5

    num_of_coins += cents

    print("{}".format(num_of_coins))


if __name__ == "__main__":

    main()
